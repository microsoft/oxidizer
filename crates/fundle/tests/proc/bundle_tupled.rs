// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

#[fundle::bundle]
struct Foo(u8);

fn main() {}
