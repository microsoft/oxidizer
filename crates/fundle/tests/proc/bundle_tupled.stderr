error: fundle::bundle only supports structs with named fields
 --> tests/proc/bundle_tupled.rs:5:1
  |
5 | struct Foo(u8);
  | ^^^^^^^^^^^^^^^
